android/samples/transparent-view/app/proguard-rules.pro

```diff
@@ -0,0 +1,21 @@
+# Add project specific ProGuard rules here.
```

Comment:
Is there any reason to keep this file?

Reply:
Just because it comes standard with every Android Studio project.